astro/src/content/docs/release-notes/index.mdx

```diff
@@ -892,7 +893,8 @@ This migration will add an index to the `identity_provider_links` table. It is n
   * Resolves [GitHub Issue #2118](https://github.com/FusionAuth/fusionauth-issues/issues/2118)
 
 ### Enhancements
-* Add default configuration for read and connect timeouts to the SMTP server configuration. This helps protect FusionAuth against an SMTP server that never closes a socket. From time to time we observed an SMTP server hold open a socket, and tie up a send thread which may block other senders.
+* Add default configuration for read and connect timeouts to the SMTP server configuration. This helps protect FusionAuth against an SMTP server that never closes a socket. From time to time we observed an SMTP server hold open a socket, and tie up a send thread which may block other senders. This includes the `mail.smtp.timeout` and 
+`mail.smtp.connectiontimeout` settings. The default value is `2000` for each. These may be overridden by navigating to the <Breadcrumb>Tenants -> Your Tenant -> Advanced</Breadcrumb> tab and adding them to the <Breadcrumb>SMTP settings</Breadcrumb> section.
```

Comment:
Thanks Dan, this is a good catch. We should have documented this earlier. 

In general, the "advanced" smtp config is documented such that it can contain any Java mail properties, and it will override anything we set by default. 

Since we do have this "advanced" section documented in the UI walk through, as well as the API doc. Should we then also document this default value in both of those location as well? 

Or maybe just in the SMTP troubleshooting section?


Our search didn't return anything useful for me, but this is what our AI bot told me about this setting:

![image](https://github.com/user-attachments/assets/67f82e40-007d-4f00-8ab1-5e6f6305e7a6)

Additional context:
- https://github.com/FusionAuth/fusionauth-issues/issues/1742#issuecomment-1454196056

Reply:
Good call, I'll update the other sections, and if we have other default SMTP config, I'll add it there.

Reply:
Updated the tenant API and the core concepts section with the default values. There was one other default value but it was added before 1.36.5 (when you did the big merge between the fusionauth-app and fusionauth-api projects) so I figured it was okay to not tag it with a 'since' note.